tensorflow/python/ops/rnn.py

```diff
@@ -126,7 +126,8 @@ def rnn(cell, inputs, initial_state=None, dtype=None,
       state = initial_state
     else:
       if not dtype:
-        raise ValueError("If no initial_state is provided, dtype must be.")
+        raise ValueError("If no initial_state is provided, "
+                           "dtype must be specified")
```

Comment:
alignment is messed up here -- this seems unrelated to this commit, maybe put in a different PR?